.circleci/config.yml

```diff
@@ -233,22 +188,45 @@ jobs:
 
   # this build runs on a fixed machine, due to a storage need
   # nothing about it necessitates the machine itself, other than the need for more disk.
-  platforms-build-gpu:
-    machine:
-      enabled: true
-      docker_layer_caching: true
-      resource_class: medium
-      image: ubuntu-2004:202101-01
-    steps:
-      - platforms-build-steps-gpu
-
-  # this build runs on a fixed machine, due to a storage need
-  # nothing about it necessitates the machine itself, other than the need for more disk.
-  platforms-build-cpu:
+  platforms-build:
+    parameters:
+      lite:
+        type: string
+      platform:
+        type: string
+      target:
+        type: string
     docker:
       - image: redisfab/rmbuilder:6.2.1-x64-buster
     steps:
-      - platforms-build-steps-cpu
+      - abort_for_docs
+      - abort_for_noci
+      - early_return_for_forked_pull_requests
```

Comment:
Can we group these doggies together, in a command like `early-aborts` or something?

Reply:
Yes, but it's out of scope for this change. I'll batch it in something else. There's a branch open locally where that's in progress.